serverless/serverless
# Remove variable syntax regex character ranges

## What did you implement

Closes #6559 and #6997. Expand regex for variable syntax by removing fixed character ranges thus allowing non ascii characters as valid fallback values.

## How can we verify it

Checkout to `dea5ecc2a0406bc46eb38f6cb8dfc17363d90498` and run `lib/classes/Variables.test.js::fallback` and `lib/plugins/print/print.test.js::should resolve fallback` tests. Both should fail. Checkout to HEAD, tests should pass.

Given a `serverless.yml`
<pre>
service: variables

provider:
  name: aws
  runtime: nodejs12.x
  profile: dev
  region: eu-central-1
  stage: dev
  environment:
    HELLO: world+
    DEFAULT: ${env:unused, '++++++'}
    JAPANESE: ${env:unused, 'システム管理者'}

functions:
  hello:
    handler: handler.hello

</pre>

With `hello` content:
<pre>
'use strict';

module.exports.hello = async event => {
  return {
    statusCode: 200,
    body: {
        message: 'Go Serverless v1.0! Your function executed successfully!',
        hello: process.env.HELLO,
        plus: process.env.DEFAULT,
        jap: process.env.JAPANESE
    },
  };
};
</pre>
When invoking it should return resolved values.

## Todos


**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** It should be backward compatible so no.
